FIAT/finite_element.py

```diff
@@ -239,11 +239,9 @@ def entity_support_dofs(elem, entity_dim):
 
     result = {}
     for f in elem.entity_dofs()[entity_dim].keys():
-        entity_transform = ref_el.get_entity_transform(entity_dim, f)
-        points = list(map(entity_transform, quad.get_points()))
-
         # Integrate the square of the basis functions on the facet.
-        vals = numpy.double(elem.tabulate(0, points)[(0,) * dim])
+        entity = (entity_dim, f)
+        vals = numpy.double(elem.tabulate(0, quad.get_points(), entity)[(0,) * dim])
```

Comment:
After testing without this change, everything appears to be working just fine. I will remove the commit.